move-vm-backend-common/tests/assets/move-projects/signer_scripts/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "signer_scripts"
```

Comment:
True that. We can try to take care of such things earlier in future projects/products.

> maybe we can harmonize this in the future

This - since this has low priority now on my task list 😄